src/cljdoc/util/fixref.clj

```diff
@@ -57,25 +56,42 @@
       (str from-uri-map anchor)
       (str scm-base root-relative))))
 
-(defn fix-image
-  [file-path src {:keys [scm-base]}]
-  (let [suffix (when (.endsWith src ".svg") "?sanitize=true")]
+(defn- scm-rev [scm]
+  (or (:name (:tag scm))
+      (:commit scm)))
+
+(defn- local-raw-server
+  "This is overly inflexible for the first version.
+  When testing locally we point to simple static http server."
+  [scm]
+  (when (scm/fs-uri (:url scm)) "http://localhost:9090/"))
+
+(defn- raw-base-url [scm]
+  (or (local-raw-server scm)
+      (str (:url scm)
+           "/raw/"
+           (scm-rev scm) "/")))
+
+(defn- fix-image
+  [file-path src scm ]
+  (let [raw-base-url (raw-base-url scm)
```

Comment:
If we only need the `scm` arg to determine `raw-base-url` we could also just pass `(raw-base-url scm)` as an argument to this function. This might make it a little easier to read as well.

Reply:
Yeah, good point!  Will address.